src/PointwiseFunctions/AnalyticData/GrMhd/FukaInitialData.hpp

```diff
@@ -0,0 +1,169 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <memory>
+#include <mutex>
+
+#include "DataStructures/Tensor/EagerMath/DotProduct.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/NumericInitialData.hpp"
+#include "IO/External/InterpolateFromFuka.hpp"
+#include "Options/String.hpp"
+#include "PointwiseFunctions/AnalyticData/GrMhd/AnalyticData.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "PointwiseFunctions/Hydro/SpecificEnthalpy.hpp"
+#include "PointwiseFunctions/Hydro/Tags.hpp"
+#include "PointwiseFunctions/InitialDataUtilities/InitialData.hpp"
+#include "Utilities/Serialization/CharmPupable.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace grmhd::AnalyticData {
+
+/*!
+ * \brief Hydro initial data generated by FUKA.
+ *
+ * This class loads numerical data written out by the FUKA initial data solver.
+ */
+class FukaInitialData : public evolution::initial_data::InitialData,
+                        public evolution::NumericInitialData,
+                        public AnalyticDataBase {
+ public:
+  struct InfoFilename {
+    using type = std::string;
+    static constexpr Options::String help = {
+        "Path to the '.info' file of data produced by FUKA."};
+  };
+
```

Comment:
Should we add a comment here or in the help string that the constant electron fraction option is most of the time inconsistent with the ID, and will need to be replaced later?

Reply:
Yes, let's do that! Does the ID write out consistent a electron fraction? If not, should that be added? We've talked about this for SpEC too, and that's the plan for spec

Reply:
Neither electron fraction nor temperature are written out by FUKA. I added a note to the dox for this class and opened https://github.com/sxs-collaboration/spectre/issues/5518.